gnosis/eth/account_abstraction/user_operation.py

```diff
@@ -18,6 +23,7 @@ class UserOperation:
     https://github.com/eth-infinitism/account-abstraction/blob/v0.6.0
     """
 
+    user_operation_hash: bytes
```

Comment:
We should use both bytes or str or decide what we want to use https://github.com/safe-global/safe-eth-py/pull/800/files#diff-ba5b3a9d52831b6f1bfb7059bfae7f9a0b90835e3b6d99855adcbafa88261171R84